mtproto/utils.go

```diff
@@ -63,6 +71,28 @@ func IsWrongClientTimeError(tlOrErr any) bool {
 	return false
 }
 
+func IsMsgSeqnoTooLowError(tlOrErr any) bool {
```

Comment:
For now, it is better not to check these errors, as it is not yet known in which case they must be handled and recovered.

It won't help with https://github.com/3bl3gamer/tgclient/issues/28#issuecomment-2241944782, and according to [GramJS source](https://github.com/gram-js/gramjs/blob/9233537e172ffce6d44a09454b7c75369a846bbf/gramjs/network/MTProtoSender.ts#L858) increasing the sequence number is not the best solution anyway.